### DISCUSSION TITLE
Can't get rid of the notification

### DISCUSSION THREAD

**Discussion by LucasBS1** [LSPosed/LSPosed#2353] [Q&A]
After the latest update, this notification came up that can't be disabled.

Problem is: the shortcut is already there.
Already tried to delete it.

Framework/Manager 1.8.6 (6712)
Magisk 25.2
Android P

Adding insult to the injury: the notification is not on the same language as the manager

![Screenshot_20230121-073905](https://user-images.githubusercontent.com/86728997/213863434-3d7d5def-46bb-4ee2-9203-992be6001786.png)

![Screenshot_20230121-074003](https://user-images.githubusercontent.com/86728997/213863449-1f9447c9-dd20-4ac3-acd8-375f44c20539.png)

![Screenshot_20230121-074055](https://user-images.githubusercontent.com/86728997/213863451-b37e692c-fad0-4d34-b817-26a3f51fd119.png)

**Comment by grysor** [comment]
> After the latest update, this notification came up that can't be disabled.
> 
> Problem is: the shortcut is already there. Already tried to delete it.
> 
> Framework/Manager 1.8.6 (6712) Magisk 25.2 Android P
> 
> Adding insult to the injury: the notification is not on the same language as the manager
> 
> ![Screenshot_20230121-073905](https://user-images.githubusercontent.com/86728997/213863434-3d7d5def-46bb-4ee2-9203-992be6001786.png)
> 
> ![Screenshot_20230121-074003](https://user-images.githubusercontent.com/86728997/213863449-1f9447c9-dd20-4ac3-acd8-375f44c20539.png)
> 
> ![Screenshot_20230121-074055](https://user-images.githubusercontent.com/86728997/213863451-b37e692c-fad0-4d34-b817-26a3f51fd119.png)




Same problem for me :(
https://github.com/LSPosed/LSPosed/issues/2355
https://github.com/libxposed/helper/issues/2
